tests/FSharp.Core.UnitTests/FSharp.Core.UnitTests.fsproj

```diff
@@ -5,14 +5,29 @@
   <PropertyGroup>
     <TargetFrameworks>net46;netcoreapp2.0</TargetFrameworks>
     <OutputType>Library</OutputType>
+
+    <AssemblyName>FSharp.Core.UnitTests</AssemblyName>
+    <PackageId>Microsoft.FSharp.Core.UnitTests</PackageId>
+    <PackageVersion>$(FSCoreUnitTestsPackageVersion)</PackageVersion>
+    <PreRelease>true</PreRelease>
+
     <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
-    <UnitTestType>nunit</UnitTestType>
+    <UnitTestType>xunit</UnitTestType>
+    <IsTestProject>true</IsTestProject>
+    <IsPackable>true</IsPackable>
 
     <!-- must be false so FsCheck can be loaded -->
     <SignAssembly>false</SignAssembly>
+    <TailCalls>true</TailCalls>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants  Condition="'$(UnitTestType)' == 'xunit'">$(DefineConstants);XUNIT</DefineConstants>
```

Comment:
You're explicitly setting `$(UnitTestType)` to `xunit` a few lines up, so this section doesn't really do anything.